base/markdown/render/plain.jl

```diff
@@ -37,6 +37,46 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, md::Table)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
```

Comment:
So the issue here is that length is the length of the array of components rather than the total length of the strings. I could define length too for these - I didn't know the purpose of plain tbh (is it a neat/aligned representation of the tables or just one that is valid markdown - that can be parsed back)?